devito/operator/profiling.py

```diff
@@ -367,21 +376,23 @@ class AdvisorProfiler(AdvancedProfiler):
     _ext_calls = [_api_resume, _api_pause]
 
     def __init__(self, name):
-        self.path = locate_intel_advisor()
-        if self.path is None:
-            self.initialized = False
+        if self._attempted_init:
+            return
+
+        super().__init__(name)
+
+        path = get_advisor_path()
+        if path:
+            self.add_include_dir(path.joinpath('include').as_posix())
```

Comment:
as explained above, this should just be `self._include_dirs.append(...)`

Reply:
yes